science/viking/files/autogen.sh

```diff
@@ -7,40 +7,32 @@ test -n "$srcdir" || srcdir=.
 olddir=`pwd`
 cd "$srcdir"
 
-INTLTOOLIZE=`which intltoolize`
+INTLTOOLIZE=`command -v intltoolize`
 if test -z $INTLTOOLIZE; then
         echo "*** No intltoolize found, please install the intltool package ***"
         exit 1
 fi
 
-GTK_DOC=`which gtkdocize`
+GTK_DOC=`command -v gtkdocize`
 if test -z $GTK_DOC; then
         echo "*** No gtkdocize found, please install the gtk-doc-tools package ***"
         exit 1
 fi
 
-GNOME_DOC=`which gnome-doc-prepare`
-if test -z $GNOME_DOC; then
-        echo "*** No gnome-doc-prepare found, please install the gnome-doc-utils package ***"
-        exit 1
-fi
-
-AUTORECONF=`which autoreconf`
+AUTORECONF=`command -v autoreconf`
 if test -z $AUTORECONF; then
         echo "*** No autoreconf found, please install it ***"
         exit 1
 fi
 
-if test -z `which autopoint`; then
+if test -z `command -v autopoint`; then
         echo "*** No autopoint found, please install it ***"
         exit 1
 fi
 
-gnome-doc-prepare --automake --copy --force || exit $?
 gtkdocize --copy || exit $?
 autopoint --force || exit $?
-AUTOPOINT='intltoolize --automake --copy --force' autoreconf --force --install --verbose || exit $?
+AUTOPOINT='intltoolize --automake --copy' autoreconf --force --install --verbose || exit $?
 
 cd "$olddir"
 test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
-
```

Comment:
Why remove the last line in the patch?

Reply:
They are identical

Reply:
This was a superfluous empty line, that apparently was removed when autogen.sh was updated.